modules/custom/openy_session_instance/src/SessionInstanceManager.php

```diff
@@ -67,8 +60,7 @@ class SessionInstanceManager implements SessionInstanceManagerInterface {
   /**
    * Constructor.
    */
-  public function __construct(EntityTypeManagerInterface $entity_type_manager, QueryFactory $entity_query, LoggerChannelFactoryInterface $logger_factory, ConfigFactoryInterface $configFactory) {
-    $this->entityQuery = $entity_query;
+  public function __construct(EntityTypeManagerInterface $entity_type_manager, LoggerChannelFactoryInterface $logger_factory, ConfigFactoryInterface $configFactory) {
```

Comment:
Usage of `$this->entityQuery` should be replaced with `$this->entityTypeManager` as well. The class is broken now.